### PR TITLE
split coordinator and node traces

### PR DESCRIPTION
**Pull request by tbg**
[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/2930)

### CODE REVIEW

File: acceptance/localcluster/localcluster.go [hunk 1]
```diff
@@ -24,6 +24,7 @@ import (
 	"io"
 	"io/ioutil"
 	"os"
+	"path"
```
Comment (tamird) [inline acceptance/localcluster/localcluster.go]: i think this fails the lint

Comment (tschottdorf) [inline acceptance/localcluster/localcluster.go]: fixed.

File: util/tracer/tracer.go [hunk 2]
```diff
@@ -218,19 +225,20 @@ var dummyTracer = &Tracer{
...
 		t = dummyTracer
 	}
-	return t.newTrace(tracee.TraceID(), tracee.TraceName())
+	return t.newTrace(family, tracee.TraceID(), tracee.TraceName())
 }
 
-func (t *Tracer) newTrace(id, name string) *Trace {
-	nt := ntrace.New("req", name)
+func (t *Tracer) newTrace(family string, id, name string) *Trace {
+	nt := ntrace.New(family, name)
```
Comment (tschottdorf) [inline util/tracer/tracer.go]: this is really what the plumbing is for.

File: acceptance/localcluster/localcluster.go (hunk 1 above)
Comment (bdarnell) [inline acceptance/localcluster/localcluster.go]: Now that we have a legitimate use case for `"path"` I'd say `make check` is mistaken here, but either way works.

Comment (tamird) [inline acceptance/localcluster/localcluster.go]: yeah, i think i agree with you. also indifferent.

**Comment by tamird** [comment]
LGTM

**Pull request merged**